README.md

```diff
@@ -45,3 +45,30 @@ PHP >= 8.0.7
 $ composer install
 $ php -S localhost:8080 -t public/
 ```
+
+## Deployment
+
+### Environments
+
+There are 3 targeted environment at the moment:
+
+* **Integration**: integration-help.prestashop-project.org
+* **Préproduction**: preprod-help.prestashop-project.org
+* **Production**: help.prestashop-project.org
+
+Those edge URLs are hosted at the Cloudflare level, proxyfying the origin GCP Storage.
+
+## Architecture
+
+![alt text](pics/architecture.png "Build & Refresh Workflow")
```

Comment:
@matks there is no gcp bucket for this project. The content is generated on-the-fly when a call is made and cached by cloudflare for 24h (if I recall).
There is already a documentation in the folder [.cloud](https://github.com/PrestaShop/contextual-help-api/tree/main/.cloud), maybe we just need to put it in the root folder?